boards/DIM/Inc/App/App_RegenPaddle.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <stdint.h>
+#include "App_ErrorCode.h"
+
+/**
+ * Allocate and initialize a regen paddle
+ * @param get_paddle_position A function that can be called to get the paddle
+ *                            position as a percentage, where 0% means not
+ *                            pressed at all and 100% means pressed all the way
+ * @param lower_deadzone The paddle position threshold of lower deadzone
+ * @param upper_deadzone The paddle position threshold of upper deadzone
```

Comment:
This is a bit ambiguous. Do we ignore values above this? Is this value mapped to 100%? Same goes for the lower deadzone.

Reply:
Good point :+1: I rephrased it so let me know if it's still ambiguous. Also clarified that paddle position not in the range of 0 - 100 is invalid.